x/emissions/types/keys.go

```diff
@@ -51,5 +51,14 @@ var (
 
 	// BallotMaturityBlocks is amount of blocks needed for ballot to mature
 	// by default is set to 100
-	BallotMaturityBlocks = 100
+	BallotMaturityBlocks = 100 // approximately 9-10 minutes
+	// PendingBallotsBufferBlocks is a buffer number of blocks
+	//(in addition to BallotMaturityBlocks)
+	// that we use only for pending ballots before deleting them
+	PendingBallotsBufferBlocks = SecondsToBlocks(60 * 60 * 24 * 10) // 10 days
 )
+
+// SecondsToBlocks converts seconds to blocks.It assumes that block time is 6 seconds
+func SecondsToBlocks(secs int64) int64 {
```

Comment:
We don't need this function, let's compute value manually and put the number of blocks in the constant directly

Reply:
Imo this makes it more readable , we normally discuss things like this in terms of `time` and not `blocks`

Reply:
You just need to say in comment how it has been computed, it's straightforward to check